### ISSUE TITLE
Clarity issue: #change and the SQL before it are the same

### ISSUE THREAD

**Issue by KatieLars** [learn-co-curriculum/mechanics-of-migrations#45]
Hi! I found the equivalency between the SQL used to create tables (described before #change) and #change was not stated very clearly. I initially interpreted it as saying that create_table was the name of the method you just created with SQL, rather than a method that comes with ActiveRecord::Migration. Also, when inheriting from ActiveRecord::Migration, I was prompted to specify the version of Rails it was using (ActiveRecord::Migration[5.0] for it to work. I use a local environment with Atom

**Comment by curiositypaths** [comment]
Thanks for reporting this issue. We're now aware of it and are tracking it in
#52. We will add this to our internal work queue and
notify you when it is resolved.

We're closing this issue but invite you to follow progress at that URL.

-- Flatiron Curriculum Team

**Comment by maxwellbenton** [comment]
Hi @KatieLars,

Thank you for providing this feedback. We've updated this lesson's readme and have added content to the table creation section to better explain the transition between using SQL directly and using migrations. We've also updated this lesson's content regarding Active Record versioning on migrations, and believe this issue should be resolved. 

Thank you for helping us ensure that those who follow in your path will not encounter the same problems that you faced. 💙
